src/containers/legacy-namespaces/legacy-namespaces.tsx

```diff
@@ -53,14 +53,16 @@ class LegacyNamespaces extends React.Component<
   }
 
   componentDidMount() {
+    const p: any = {};
     const thisQS = window.location.search;
     const urlParams = new URLSearchParams(thisQS);
-    const page = parseInt(urlParams.get('page')) || 1;
-    const page_size = parseInt(urlParams.get('page_size')) || 10;
-    const order_by = urlParams.get('order_by') || 'name';
-    const keywords = urlParams.get('keywords') || null;
+    p.page = parseInt(urlParams.get('page')) || 1;
+    p.page_size = parseInt(urlParams.get('page_size')) || 10;
+    p.order_by = urlParams.get('order_by') || 'name';
+    p.keywords = urlParams.get('keywords') || null;
 
-    LegacyNamespaceAPI.list({
+    this.updateParams(p);
```

Comment:
```suggestion
    this.updateParams({
      page: parseInt(urlParams.get('page'), 10) || 1,
      page_size: parseInt(urlParams.get('page_size'), 10) || 10,
      order_by: urlParams.get('order_by') || 'name',
      keywords: urlParams.get('keywords') || null,
    });
```

and remove the commented out code please :)